tests/distributed/test_distributed_sampling.py

```diff
@@ -220,3 +256,5 @@ def test_rpc_in_subgraph():
         check_rpc_sampling_shuffle(Path(tmpdirname), 2)
         check_rpc_sampling(Path(tmpdirname), 2)
         check_rpc_sampling(Path(tmpdirname), 1)
+        check_rpc_find_edges(Path(tmpdirname), 2)
+        check_rpc_find_edges(Path(tmpdirname), 1)
```

Comment:
is the test here the same as the one in test_dist_graph_store?

Reply:
2 will create partitions as local and remote.